style.css

```diff
@@ -313,7 +313,21 @@ form {
   justify-content: space-between;
   flex-direction: row;
 }
-
+.footer__drop-container {
+  display: flex;
+  justify-content: flex-start;
+  margin: 0 1% 2rem 1%;
+  border-top: 1px solid #98989c;
+}
+.footer__dropdown {
+  margin: 2rem 1.5% 2rem 1.5%;
+  width: 10rem;
+  border: none;
+  background: #f7f8f8;
+  font-family: "Poppins", sans-serif;
```

Comment:
Better to add this to the actual 'select' element, i.e `select {}` then you won't need to declare it for every select

Reply:
Wrote in wrong branch name accidentally created new pr